src/features/profile/DangerCard.js

```diff
@@ -95,6 +96,17 @@ const DangerCard = () => {
         >
           {deleteAllEntriesText}
         </Button>
+
+        <Button
+          // true if entrie.length > 0
+          disabled={disabled}
+          // alert "Clearing tags" when clicked. (This will later be replaced with proper functionality).
+          variant="outlined"
+          color="primary"
+          // onClick={clearMensenTags}
```

Comment:
Create a `handleClearMensesTags` where you do all the setPending stuff and error handling. Take a look at `handleDeleteAllEntries`. (No need to have "confirm" alert here as in delete all entries).